articles/virtual-machines/automatic-vm-guest-patching.md

```diff
@@ -239,7 +239,9 @@ Set-AzVMOperatingSystem -VM $VirtualMachine -Windows -ComputerName $ComputerName
 Use the [Set-AzVMOperatingSystem](/powershell/module/az.compute/set-azvmoperatingsystem) and [Update-AzVM](/powershell/module/az.compute/update-azvm) cmdlet to enable automatic VM guest patching on an existing VM.
 
 ```azurepowershell-interactive
-Get-AzVM -VM $VirtualMachine -Windows -ComputerName $ComputerName -Credential $Credential
+$VirtualMachineName='myVM'
+$ResourceGroupName='myResourceGroup'
+$VirtualMachine=Get-AzVM -ResourceGroupName $ResourceGroupName -Name $VirtualMachineName
```

Comment:
Thanks for noticing the syntax error and submitting a PR. I'd like to see it updated to conform to syntax of the other Get's within the documentation:

$VirtualMachine = Get-AzVM -ResourceGroupName "myResourceGroup" -Name "myVM"